internal/appv2/patches.go

```diff
@@ -198,6 +198,23 @@ func _patchService(service map[string]any) (map[string]any, error) {
 		}
 	}
 
+	if rawInternalPort, ok := service["internal_port"]; ok {
+		switch cast := rawInternalPort.(type) {
+		case string:
+			n, err := strconv.Atoi(cast)
+			if err != nil {
+				return nil, fmt.Errorf("Can not convert internal_port '%s' to integer: %w", cast, err)
+			}
+			service["internal_port"] = n
+		case float64:
+			service["internal_port"] = int(cast)
+		case int64:
+			service["internal_port"] = int(cast)
+		default:
+			return nil, fmt.Errorf("Unknown type for internal_port number: %T", cast)
+		}
```

Comment:
[question/suggestion] would the type still be **int64/float64** if the person was running flyctl on a 32 bit machine? (idk for sure, im just wondering)

What do you think about something like 👇 ?
```suggestion
		switch cast := rawInternalPort.(type) {
		case string:
			n, err := strconv.Atoi(cast)
			if err != nil {
				return nil, fmt.Errorf("Can not convert internal_port '%s' to integer: %w", cast, err)
			}
			service["internal_port"] = n
		default:
		        port, ok := rawInternalPort.(int)
		        if !ok {
		            return nil, fmt.Errorf("Unknown type for internal_port number: %T", cast)
		        }
		        service["internal_port"] = port
		}
```

Reply:
This solution looks really clean, but doesn't work unfortuanetly. It works great for string ports, but when I try it with a regular int port I get `Error invalid config: Unknown type for internal_port number: int64`

Reply:
day by day, my reputation as a liar goes up. 🥱